pkg/controller/kvcache/kvcache_controller.go

```diff
@@ -561,6 +561,13 @@ func (r *KVCacheReconciler) reconcileDeployment(ctx context.Context, kvCache *or
 						},
 					},
 					Affinity: &affinity,
+					Tolerations: []corev1.Toleration{
+						{
+							Key:      "nvidia.com/gpu",
```

Comment:
this is pretty common taint and there's no harm to add the tolerations. We should be safe to add it.